src/main/resources/META-INF/plugin.xml

```diff
@@ -64,6 +64,7 @@
 
     <change-notes><![CDATA[
       <ul>
+        <li>10.3 - 9 new C++ MISRA 2023 rules. Remove support for Node.JS 14. Support TypeScript 5.3. 5 new rules for JavaScript</li>
```

Comment:
It could be worth mentioning that we now support rules for the Jakarta namespace, it was reported as an issue for some users